### ISSUE TITLE
No element videoconvertscale for Ubuntu 22.04 build (gstreamer 1.20.3)

### ISSUE THREAD

**Issue by RKelln** [brunoherbelin/vimix#105]
Blargh, was excited to try the screen capture of a window but got this:

```
0028   Warning - Stream 698277812802262680 Could not construct pipeline: no element "videoconvertscale"
```

I checked gstreamer and its not there, but couldn't figure out which version of gstreamer it is a part of, I'm on GStreamer 1.20.3 (Ubuntu 22.04). I suppose can can change that to just `videoconvert` for now?

Is it possible to release an update for the flatpak (mine is still at 0.8.0)?

**Comment by brunoherbelin** [comment]
Sorry for that : indeed this was an undocumented change in gstreamer and 0.8.1 has it fixed.

If you are not afraid of a few command lines, creating a custom flatpack is actually not that difficult:
https://github.com/brunoherbelin/vimix/blob/master/flatpak/README.md

I am working on the 0.8.1 release, but maybe will just do a patch 0.8.0b... I'll let you know

**Comment by RKelln** [comment]
Cool, yeah I was thinking that you might be able to get a newer gstreamer in a flatpack build. I figured out that I wasn't on the latest locally and saw the `videoconvertscale` revert, so I'm building latest now, that should fix my problem for now.

**Comment by brunoherbelin** [comment]
Fixed in 0.8.1
